### ISSUE TITLE
AutoCorrect Feature for Blank Line Before Declarations

### ISSUE THREAD

**Issue by qichengri** [detekt/detekt#6800]
## Expected Behavior
Auto Correct Feature for `Blank Line Before Declarations`
https://pinterest.github.io/ktlint/0.50.0/rules/experimental/#blank-line-before-declarations

## Current Behavior
`SpacingBetweenPackageAndImports` is helpful for validate `Blank Line Before Declarations`.
But looks like there is not related formating/autoCorrect feature for `Blank Line Before Declarations`

### As is
```kotlin
package com.demo
import java.time.Duration
class DemoClass {
}
```
### To be
```kotlin
package com.demo
// If the formating name is BlankLineBeforeDeclarations, this blank line is optional, depends on how we define the name of the formating.
import java.time.Duration

class DemoClass {
}
```

**Comment by 3flex** [comment]
The wrapper for this rule was added to detekt-formatting in https://github.com/detekt/detekt/pull/6253 but is not yet in a released version. You may be able to use snapshot version of detekt-formatting in your project until there's an updated release.
